Apres_Ski_API/admin.py

```diff
@@ -0,0 +1,18 @@
+from django.contrib import admin
+from Apres_Ski_API.models.user import User
+from Apres_Ski_API.models.restaurant import Restaurant
+from Apres_Ski_API.models.hour import Hour
+from Apres_Ski_API.models.happyhour import HappyHour
+from Apres_Ski_API.models.photo import Photo
+from Apres_Ski_API.models.engagement import Engagement
+from Apres_Ski_API.models.comment import Comment
+from Apres_Ski_API.models.lift import Lift
```

Comment:
When we created our test_api we wanted to test out one-one, one-many, & many-many relationships in Django. We were able to take what we learned there and quickly create all the tables (MVP and extensions).

We can delete the non-mvp tables. We just thought it would be nice to include them so if FE would have access to all the information.

Reply:
Talking with FE about this change. We would like to move the `engagement` table to our MVP. This would leave `photo` and `comment` as the only non-MVP tables.

We would like to keep them in as they are created and functioning. This will also let us to just focus on Pytest (as opposed to creating another PR on demand for when FE needs photos or comments).

Totally understand if that is not ok. Please let us know.

Reply:
I think keeping this work at this point is fine, rather than recreating it later. I think it's just a good process to consider when talking about waterfall vs agile processes and which you're following. I also totally understand wanting to play with all those relationships within this new framework so I can see now why these are here.